### PR TITLE
Add a CLI to the analyzer

### PR DESCRIPTION
**Pull request by sauyon**

### CODE REVIEW

**Comment by sauyon** [comment]
Relies on #16

File: pipeline/analyzer/analyzer.go [hunk 1]
```diff
@@ -1,20 +1,110 @@
...
+	var err error
 	flag.Parse()
 	cfg = config.Load(*cfgFile)
+	db, err = openDb()
+	if err != nil {
+		log.Fatal("Failed to open a connection to the database %s", err.Error())
+	}
+}
+
+func analyze(app *App) error {
```
Comment (deanott) [inline pipeline/analyzer/analyzer.go:29]: Remove analyzer and put main into own main.go starter

**Comment by deanott** [comment]
This is great and runs 👍 Left a comment for a potential restructure we discussed

**Review by deanott — APPROVED** [review]
Needed to approve in this section 💯

**Pull request merged**